README.md

```diff
@@ -19,3 +19,42 @@ Chosen topic: Amazin Bookstore & FaaS, Microservices, Amazon Lambda
 - Milestone 2: March 22 2023
 - Group Presentation: March 26 2023
 - Milestone 3: April 5 2023
+
+### Team members
+- Phuc La (101107588)
+- Nicolas Tuttle (101105699)
+- Robell Gabriel (101108508)
+
+#### How to run the project
+1. Package Maven
+2. Run Sysc4806ProjectApplication.class
+
+#### Milestone 1
+
+In milestone 1, our team built the framework of the Amazin BookStore with many features implemented. Features created in this milestone: 
+- Create new book for the store (upload new book to the store):
+  - Bookstore owner or admin can add new book into the store.
+- Edit book (update book information such as description and inventory):
+  - Bookstore owner or admin can edit the information of the pre-existing book in the store by changing the description or update inventory manually.
+- Delete book from the store page:
+  - Bookstore owner or admin can delete a book from the store page, which completely removes it from the system.
+- Adding books into shopping cart
+  - Users can add book into their respective online shopping cart, and the book's inventory will decrease accordingly based on the number of books that the user added into their cart.
```

Comment:
the shopping cart doesn't work this way, book's in the inventory will decrease only after the user checkouts and pays for the books.

Reply:
Nice catch